examples/Gameraww/app/tns_modules/uiimage-module-loader/index.js

```diff
@@ -0,0 +1,21 @@
+export function resolve(loader, key, referer) { return key; }
+
+export function fetch(loader, key) {
+  let imageName = key.replace(/^@.*\//, '');
```

Comment:
I think that the regex should be `^@.*?\/` to make sure `@uiimage/folder/reddit-default` is matched correctly.